### PR TITLE
Feature/fix style

### PR DESCRIPTION
**Pull request by andriikamaldinov1**

### CODE REVIEW

File: open-source/accordion/accordion.component.ts [hunk 1]
```diff
@@ -35,17 +36,23 @@ import { TrackByService } from '@libraries/track-by/track-by.service';
...
-
     @Input() public lists!: IListItem[];
     @Output() public itemAccordion = new EventEmitter<number>();
     @Output() public itemInAccordion = new EventEmitter<number>();
     @ViewChildren('accordion', { read: ElementRef }) public accordion!: QueryList<ElementRef>;
+    public constructor(private activatedRoute: ActivatedRoute) {}
 
     public ngAfterViewInit(): void {
         this.openFirstAccordion();
+        this.activatedRoute.fragment.subscribe((res) => {
```
Comment (GlebChiz) [inline open-source/accordion/accordion.component.ts:50]: remove check from subscribe and add it to filter rxjs operator 

`this.activatedRoute.fragment.pipe(filter(Boolean)).subscribe((res) => {
this.chosenItem = +res;
                this.itemInAccordion.emit(+res);}`

File: open-source/accordion/accordion.component.ts [hunk 2]
```diff
@@ -35,17 +37,21 @@ import { TrackByService } from '@libraries/track-by/track-by.service';
...
-    public chosenItem = 1;
     public chosenList = 1;
+    public chosenItem = 1;
     public readonly trackByPath = inject(TrackByService).trackBy('id');
-
     @Input() public lists!: IListItem[];
     @Output() public itemAccordion = new EventEmitter<number>();
     @Output() public itemInAccordion = new EventEmitter<number>();
     @ViewChildren('accordion', { read: ElementRef }) public accordion!: QueryList<ElementRef>;
+    public constructor(private activatedRoute: ActivatedRoute) {}
```
Comment (xakeppok) [inline open-source/accordion/accordion.component.ts:47]: use inject() approach

File: open-source/accordion/accordion.component.ts [hunk 3]
```diff
@@ -35,17 +37,21 @@ import { TrackByService } from '@libraries/track-by/track-by.service';
...
 
     public ngAfterViewInit(): void {
         this.openFirstAccordion();
+        this.activatedRoute.fragment.pipe(filter(Boolean)).subscribe((res) => {
```
Comment (xakeppok) [inline open-source/accordion/accordion.component.ts:51]: what does 'res' mean? please use more readable variables

File: open-source/service/scroll.service.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root',
```
Comment (xakeppok) [inline open-source/service/scroll.service.ts:4]: dont use providedIn:root, inject it where you need it

File: open-source/accordion/accordion.component.ts [hunk 5]
```diff
@@ -35,17 +37,21 @@ import { TrackByService } from '@libraries/track-by/track-by.service';
...
+            this.chosenItem = +res;
```
Comment (xakeppok) [inline open-source/accordion/accordion.component.ts:52]: maybe use Observables and RxJs operators for that ? Also for what are you using fragment there ?

**Pull request merged**